frontend/public/models/yaml-templates.ts

```diff
@@ -693,4 +693,48 @@ spec:
     type: JenkinsPipeline
   triggers:
   - type: ConfigChange
+`).setIn([referenceForModel(k8sModels.ResourceQuotaModel), 'rq-compute'], `
+apiVersion: v1
+kind: ResourceQuota
+metadata:
+  name: compute-quota
+  namespace: test-ns
+spec:
+  hard:
+    pods: '4'
+    requests.cpu: '1'
+    requests.memory: 1Gi
+    limits.cpu: '2'
+    limits.memory: 2Gi
+    requests.nvidia.com/gpu: 4
```

Comment:
I would leave this off. I'm not sure what configuration is needed on the cluster for it to work.